codec-http2/src/main/java/io/netty/handler/codec/http2/internal/hpack/Decoder.java

```diff
@@ -295,10 +295,13 @@ public void decode(int streamId, ByteBuf in, Http2Headers headers) throws Http2E
                 default:
                     throw new Error("should not reach here state: " + state);
             }
+        }
```

Comment:
doh ... good catch. Double fail bcz I forgot to add a unit test for the decoder side of the house ... unit test focused on the encoder https://github.com/netty/netty/commit/2fd42cfc6b32dacb812c6f2ecef8522cd94718d4#diff-928e2c1c5d3a80187c2936e9a4bfced7R217